src/FullCalendar.tsx

```diff
@@ -1,32 +1,22 @@
 import * as React from 'react'
 import { Calendar, OptionsInput } from '@fullcalendar/core'
 
-export default class FullCalendar extends React.Component<OptionsInput, any> {
+const { forwardRef, useEffect, useImperativeHandle, useRef, useState } = React
```

Comment:
is it possible to change this to `import { bla, bla, bla } from React` or will JSX get confused?

Reply:
...and combine it `import * as React from 'react'` too? it'd be nice to have only one statement

Reply:
That's what I'd have done but I face this error:

`Module '"/Users/Paul/Projets/fullcalendar-react/node_modules/@types/react/index"' has no default export.`

I've never coded in typescript before and this error doesn't happen in javascript.

The only other working thing I managed to do is:

`import * as React from 'react'`
`import { forwardRef, useEffect, useImperativeHandle, useRef, useState } from 'react'`

I was hoping you guys would have a solution to improve this ugliness.

Reply:
ah yes, i remember typescript complains. in that case, it's good to just have it like this:

```js
import * as React from 'react'
import { forwardRef, useEffect, useImperativeHandle, useRef, useState } from 'react'
```